tests/jballerina-unit-test/src/test/resources/test-src/klass/class_def_test.bal

```diff
@@ -31,6 +31,22 @@ function testFooClass() {
     assertEquality(f.s, COMMA);
 }
 
+int a = 3;
+
+public class Bar {
+    int a = 4;
+    int b = a;
+
+    function init() {
+        assertEquality(self.b, 3);
```

Comment:
Why are we doing the assertion here rather than in the actual test function?

Reply:
Removed